fcsa_service_request-1.0.0-resolved.json

```diff
@@ -0,0 +1,668 @@
+{
+  "openapi": "3.0.0",
+  "info": {
+    "title": "FCSA Service Request",
+    "description": "FCSA Service Request",
+    "contact": {
+      "email": "dee@opencitieslab.org"
```

Comment:
This should rather be the email addresses of the relevant people at the City who maintain the API.

Reply:
I have placed a dump email for the meantime, until we know who will be maintaining this documentation. Is that fine?